### ISSUE TITLE
Focus jumps around in webview when scale factor > 1

### ISSUE THREAD

**Issue by CharlieHess** [electron/electron#7655]
- Electron version: 1.4.3
- Operating system: Windows 10

While hovering over certain elements focus leaps around the page. This seems to be a very recent regression; we've haven't made any changes that would cause this within our embedded page. 

Here's a [video](https://cl.ly/1B170C2G032A) of what I mean. I'll try to provide more isolating info.

**Comment by CharlieHess** [comment]
Seems the trick to reproducing this is setting DPI > 100%. I have a dual-monitor setup, but even when both are set consistently (say, 200%) it still happens. But if both are at 100% this does not happen.

**Comment by eladams** [comment]
Also in:

- Electron version: 1.4.4
- Operating system: Windows 10

last version that it worked well on was 1.3.8

**Comment by CharlieHess** [comment]
Perhaps related to `use-zoom-for-dsf` changes that went into Chrome 53?

**Comment by eladams** [comment]
Interesting @CharlieHess, Google came out with this:
https://bugs.chromium.org/p/chromium/issues/detail?id=613414
https://github.com/atom/atom/issues/2833
https://github.com/Microsoft/vscode/issues/1896

**Comment by Thomas101** [comment]
@CharlieHess did you happen to see this on a Lenovo laptop by any chance?

**Comment by kwonoj** [comment]
@Thomas101 I'm using lenovo x1 yoga (2560x1600, 150%) and seeing this constantly.

**Comment by Thomas101** [comment]
I've had some lenovo yoga users report issues with highlighting text and resizing textareas. The videos I've [been sent](https://github.com/Thomas101/wmail/issues/334#issuecomment-251073247) seem to be similar to the video @CharlieHess uploaded. Was not able to reproduce the issues on non-lenovo laptops though.

I've attached the simplest example I could put together... [wmail-lenovo-test copy.zip](https://github.com/electron/electron/files/548229/wmail-lenovo-test.copy.zip) and also a [prebuilt version](https://drive.google.com/open?id=0B--sEKXlbUTRZ09FYVhYSDlwaTQ) ... a video of the bug that came through for me was [uploaded here](https://sendvid.com/5bq2ftax).

I'd hate to hijack your issue so if it all works for you I'm happy to open elsewhere but on the face of it looks like it might be related :-/

**Comment by CharlieHess** [comment]
@Thomas101 This issue happens on every machine I've tried. 😞

**Comment by eladams** [comment]
@Thomas101 and @CharlieHess I'm using version 1.3.8 currently for my Windows build in order avoid this issue.

**Comment by andymartinwork** [comment]
We are getting this on our app. Every windows machine with a zoom factor > 100%.
It happens just like the the video in slack.

This is a serious bug which makes parts of the app unusable for some users.
I will be downgrading to 1.3.8 and pushing that out to users until this is fixed.

**Comment by MarshallOfSound** [comment]
@CharlieHess @andymartinwork @zcbenz I have reports that one of the debug builds I gave people with this line of code.

```js
app.commandLine.appendSwitch('enable-use-zoom-for-dsf', 'false');
```

Fixed the issue, is this something that should be done at an Electron level or should we attempt to find root cause?

**Comment by CharlieHess** [comment]
@MarshallOfSound yeah that fixes it for us, too. I guess we should call attention to this at the Chromium level (maybe it's already been filed?) and in the meantime add the switch to Electron.

**Comment by yuya-oc** [comment]
In my Electron app, I feel this problem was fixed by upgrading to 1.6.1. Did anyone confirm this? I'm using 150% of DPI.

**Comment by zcbenz** [comment]
I'm closing this since it seems to have been fixed in 1.6.x.
